### PR TITLE
call distinct on the start-cards to ensure they are not repeated

### PR DESCRIPTION
**Pull request by locks**
Closes https://github.com/mtgred/netrunner/issues/5691

### CODE REVIEW

File: src/clj/game/core/turns.clj [hunk 1]
```diff
@@ -1,24 +1,24 @@
 (ns game.core.turns
   (:require
-    [game.core.agendas :refer [update-all-advancement-requirements]]
```
Comment (NoahTheDuke) [inline src/clj/game/core/turns.clj:3]: Let's limit the changes to non-cosmetic stuff

File: src/clj/game/core/turns.clj [hunk 2]
```diff
@@ -70,8 +70,8 @@
 
   (let [phase (if (= side :corp) :corp-phase-12 :runner-phase-12)
         start-cards (filter #(card-flag-fn? state side % phase true)
-                            (concat (all-active state side)
-                                    (remove facedown? (all-installed state side))))
+                            (distinct (concat (all-active state side)
+                                    (remove facedown? (all-installed state side)))))
```
Comment (NoahTheDuke) [inline src/clj/game/core/turns.clj:74]: Align this with the `all-active` call or move it to the above line.

File: src/clj/game/core/turns.clj [hunk 3]
```diff
@@ -1,24 +1,24 @@
...
+   [game.core.hand-size :refer [hand-size]]
+   [game.core.ice :refer [update-all-ice update-breaker-strength]]
+   [game.core.moving :refer [move]]
+   [game.core.say :refer [system-msg]]
+   [game.core.toasts :refer [toast]]
+   [game.core.update :refer [update!]]
+   [game.core.winning :refer [flatline]]
+   [game.macros :refer [continue-ability req wait-for]]
+   [game.utils :refer [abs dissoc-in quantify]]
+   [clojure.string :as string])
```
Comment (NoahTheDuke) [inline src/clj/game/core/turns.clj:21]: Looks like the final `)` was deleted by accident.

**Comment by NoahTheDuke** [comment]
Thanks for this fix!

File: src/clj/game/core/turns.clj (hunk 1 above)
Comment (locks) [reply src/clj/game/core/turns.clj]: Oops, overeager committing!

**Comment by locks** [comment]
fyi I was closing/opening to try and get CI to run but that didn't work :P I'm not sure how to test toasts, so if you want to you can go ahead and merge.

**Comment by NoahTheDuke** [comment]
We have no tests for toasts lol. Looks great!

**Pull request merged**